modules/services/redshift-gammastep/lib/options.nix

```
@@ -183,6 +183,8 @@ in {
     xdg.configFile.${xdgConfigFilePath}.source =
       settingsFormat.generate xdgConfigFilePath cfg.settings;
 
+    home.packages = mkIf cfg.tray [ cfg.package ];
```

```suggestion
    home.packages = [ cfg.package ];
```

Let's install it unconditionally, it seems to contain useful stuff.

cc @thiagokokada

---

Also fine by me ;-)